### PR TITLE
feat: remove markdown links

### LINKED ISSUE #594: FR: Remove markdown links in samples/generated
Feature request: Remove markdown links in samples/generated:

```
*  A list of the full names of the assets. For example:
   *  `//compute.googleapis.com/projects/my_project_123/zones/zone1/instances/instance1`.
   *  See [Resource Names](https://cloud.google.com/apis/design/resource_names#full_resource_name)
   *  and [Resource Name
   *  Format](https://cloud.google.com/asset-inventory/docs/resource-name-format)
   *  for more info.
   ```

### LINKED ISSUE #977: FR: Remove markdown links in samples/generated
Feature request: Remove markdown links in samples/generated:

```
*  A list of the full names of the assets. For example:
   *  `//compute.googleapis.com/projects/my_project_123/zones/zone1/instances/instance1`.
   *  See [Resource Names](https://cloud.google.com/apis/design/resource_names#full_resource_name)
   *  and [Resource Name
   *  Format](https://cloud.google.com/asset-inventory/docs/resource-name-format)
   *  for more info.
   ```

### PR DESCRIPTION
**Pull request by sofisl**
Fixes googleapis/google-cloud-node-core#594

### CODE REVIEW

File: templates/typescript_samples/samples/generated/$version/_util.njk [hunk 1]
```diff
@@ -25,6 +28,13 @@
 {%- endfor -%}
 {%- endmacro -%}
 
+{%- macro markDownStuff(formattedLine) %}
```
Comment (summer-ji-eng) [inline templates/typescript_samples/samples/generated/$version/_util.njk:31]: Could you rename `markDownStuff` to some name readable? 😆 
And looks like this method has not been used.

File: templates/typescript_samples/samples/generated/$version/_util.njk [hunk 2]
```diff
@@ -6,8 +6,11 @@
   /**
 {%- for line in lines %}
 {%- set trimmed = line.replace(r/\s+$/, '') -%}
-{%- if trimmed.length > 0 %}
-   * {{ trimmed.replaceAll('*/', '* /') | safe }}
+{%- if trimmed.length > 0 -%}
+{%- set regExp = r/\[(.*?)\]\(.*?\)/g -%}
```
Comment (summer-ji-eng) [inline templates/typescript_samples/samples/generated/$version/_util.njk:10]: This line is similar to line 32, is that possible we could reuse the regExp

**Review by summer-ji-eng — CHANGES_REQUESTED** [review]
Try to understand what's the motivation of removing the markdown. Maybe I miss something in chat.

**Review by bcoe — APPROVED** [review]
Left a nit, looks good to me in general though.

**Review by fhinkel — APPROVED** [review]
LGTM. Since this is all in JS files and not in TS comments, I think it's good this way.

**Review by summer-ji-eng — APPROVED** [review]

**Pull request merged**